cypress/support/generate.user.js

```diff
@@ -0,0 +1,22 @@
+import { faker } from '@faker-js/faker';
+
+function generateWorker () {
+  const randomNumber = Math.random().toString().slice(4);
+  const firstName = faker.person.firstName() + randomNumber;
```

Comment:
The `firstName` is concatenated with a random number. This might be intentional for uniqueness, but if not, consider removing the random number to keep the name realistic.